core.md

```diff
@@ -274,25 +319,25 @@ before, but with a different offset. The offset is now 16, because it
 points at the `to` point (and each point consists of two `float64`s,
 which are 8 bytes each).
 
-Unlike arrays and structs, accessing a field of primitive type does
+Unlike for arrays and structs, accessing a field of primitive type does
```

Comment:
Since we no longer have "array types" (that is, they are not type descriptors), this sentence could probably read "unlike for structs,..."